packages/dashboard-message-bus/lib/message/types.ts

```diff
@@ -0,0 +1,44 @@
+export interface Message {
+  id: number;
+  type: string;
+  payload: any;
+}
+
+export interface DashboardProviderMessage extends Message {
+  type: "dashboard-provider";
```

Comment:
can probably just call this 
```suggestion
  type: "provider";
```